public/js/dashboard.js

```diff
@@ -21,17 +21,17 @@ $( document ).ready(function() {
     	$(this).parent().closest('tr').prop("class", "selected");
 
 
-    	if ( action_type === "view" )
+    	if (action_type === "view")
     	{
     		window.location = action;
     	}
 
-    	if ( action_type === "delete" )
+    	if (action_type === "delete")
```

Comment:
Identifier 'action_type' is not in camel case.<br>Mixed double and single quotes.